src/components/appmodallogin.main.jsx

```diff
@@ -88,8 +90,10 @@ class AppModalLoginMain extends React.Component {
 
   render() {
     const { failedLogin, isLoading } = this.state;
+    const { isMobileView } = this.props;
+
     return (
-      <div className="modal" id="login-modal">
+      <div className="modal" id={`login-modal${isMobileView ? '-mobile' : ''}`}>
```

Comment:
I am seeing errors in the console related to this change, and similar error in other components as well. 

Warning: Failed prop type: The prop `isMobileView` is marked as required in `AppModalLoginMain`, but its value is `undefined`.
    in AppModalLoginMain (created by Route)
    in Route (created by withRouter(AppModalLoginMain))
    in withRouter(AppModalLoginMain) (created by AppHeaderMain)
    in AppHeaderMain (created by Root)
    in Root (created by WithAnalytics)
    in WithAnalytics (created by Route)
    in Route (created by withRouter(WithAnalytics))
    in withRouter(WithAnalytics) (created by AppWithRouter)
    in Router (created by BrowserRouter)
    in BrowserRouter (created by AppWithRouter)
    in AppWithRouter

Reply:
Fixed.